### ISSUE TITLE
Indentation warning on keyframes

### ISSUE THREAD

**Issue by s7dhansh** [brigade/scss-lint#805]
```
@keyframes animateDot1 {
	0% { transform: rotate(0deg) translateX(-$animation-radius); }
	25% { transform: rotate(180deg) translateX(-$animation-radius); }
	75% { transform: rotate(180deg) translateX(-$animation-radius); }
	100% { transform: rotate(360deg) translateX(-$animation-radius); }
}
```

causes an indentation warning: "Mixed tabs and spaces".

Can someone explain it to me?

**Comment by sds** [comment]
Hey @s7dhansh,

The error message here is pretty clear. You need to configure your editor to convert tabs to spaces in order for the `Indentation` linter to be able to correctly determine indentation of source code lines.

If this is noisy, feel free to disable it in your `.scss-lint.yml`:

```yaml
linters:
  Indentation:
    enabled: false
```
